lib/host/wasi_nn/wasinnmodule.cpp

```diff
@@ -3,11 +3,21 @@
 
 #include "host/wasi_nn/wasinnmodule.h"
 #include "host/wasi_nn/wasinnfunc.h"
+#include "common/errcode.h"
+#include "runtime/hostfunc.h"
+#include "runtime/importobj.h"
+#include "spdlog/sinks/stdout_color_sinks.h"
 
 namespace WasmEdge {
 namespace Host {
 
 WasiNNModule::WasiNNModule() : ImportObject("wasi_ephemeral_nn") {
+  spdlog::set_level(spdlog::level::info);
+  spdlog::stdout_color_mt("WasiNN");
```

Comment:
I think we should use the spdlog settings in the WasmEdge and not to modify here.